### ISSUE TITLE
KeyError "car"

### ISSUE THREAD

**Issue by ian-r-rose** [CityOfLosAngeles/count-machine#20]
Things seem to fail when no "Valid objects" are selected.

To reproduce:
1. Load a video
1. Don't select any objects in the "Valid Objects" selector
1. Click "Generate"

I get the following error:
```pytb
Counting for object id:1 as car
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/ian/anaconda3/envs/count-machine/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/home/ian/anaconda3/envs/count-machine/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/c/Users/itacid457/repos/count-machine/GUI/controller/main_controller.py", line 76, in update_video_canvas
    tracker.trackObjects(parser)
  File "/mnt/c/Users/itacid457/repos/count-machine/development/tracking/object_tracker.py", line 389, in trackObjects
    self.predictMovingObjectsNewPosition(matrix_h,cur_frame_available_moving_objects,detected_objects)
  File "/mnt/c/Users/itacid457/repos/count-machine/development/tracking/object_tracker.py", line 497, in predictMovingObjectsNewPosition
    self.drawBoxesOnFrame(obj_m)
  File "/mnt/c/Users/itacid457/repos/count-machine/development/tracking/object_tracker.py", line 601, in drawBoxesOnFrame
    plot_one_box(img_ori, [x0, y0, x1, y1], label=mess, color=self.color_table[mess])
KeyError: 'car'
```
cc @vahedi25411

**Comment by vahedi25411** [comment]
The error fixed on [this](https://github.com/CityOfLosAngeles/count-machine/commit/d1cef2d5ab8a834b3fb1462984e52a12a8a8c72d) commit.

**Comment by ian-r-rose** [comment]
Fixed  by d1cef2d5ab8a834b3fb1462984e52a12a8a8c72d
